component/azstorage/block_blob.go

```diff
@@ -806,24 +804,24 @@ func (bb *BlockBlob) calculateBlockSize(name string, fileSize int64) (blockSize
 	}
 
 	// If bufferSize <= BlockBlobMaxUploadBlobBytes, then Upload should be used with just 1 I/O request
-	if fileSize <= azblob.BlockBlobMaxUploadBlobBytes {
+	if fileSize <= blockblob.MaxUploadBlobBytes {
 		// Files up to 256MB can be uploaded as a single block
-		blockSize = azblob.BlockBlobMaxUploadBlobBytes
+		blockSize = blockblob.MaxUploadBlobBytes
 	} else {
 		// buffer / max blocks = block size to use all 50,000 blocks
-		blockSize = int64(math.Ceil(float64(fileSize) / azblob.BlockBlobMaxBlocks))
+		blockSize = int64(math.Ceil(float64(fileSize) / blockblob.MaxBlocks))
```

Comment:
We shall validate any of these values have changed or not. Existing code: < 256MB single shot upload, > 256MB divide into blocks. If user gives block size use that otherwise calculate in code assuming 16MB default.

Reply:
The value of the constants are same before and after track2 migration.